src/components/ModalComponent.tsx

```diff
@@ -19,6 +20,8 @@ interface IProps {
 }
 
 const ModalComponent = (props: IProps & RouteComponentProps) => {
+  const { socketSend } = useContext(SocketContext);
```

Comment:
Hvorfor har du importert socketSend her? Den skal invokes i callback isåfall, så den skal ikke trenges her, men i parent komponenten hvor du sender inn callbacks som props til ModalComponent

```suggestion

```